vagrant/provisioning/roles/solr/tasks/main.yml

```diff
@@ -180,6 +180,23 @@
     mode: '0644'
   when: solr_version is version('8.8.2', '>=')
 
+- name: Add Strict-Transport-Security header rule to jetty.xml
+  become: yes
+  blockinfile:
+    path: /opt/app/arkcase/app/solr/server/etc/jetty.xml
```

Comment:
This path should refer to a variable, arkcase_root if I recall correctly: "{{ arkcase_root }}/app/solr/server/etc/jetty.xml".  Many examples of this in other roles.